<?xml version="1.0" encoding="utf-8"?>
<TestRecord xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Results>
    <UnitTestResult>
      <TestDate>2012-05-12T15:00:36</TestDate>
      <Status>Success</Status>
      <TotalFailures>0</TotalFailures>
      <TotalSuccess>2</TotalSuccess>
      <TotalIgnored>0</TotalIgnored>
      <Time />
      <Message>Test successful

Execution time: 0.80ms</Message>
    </UnitTestResult>
    <UnitTestResult>
      <TestDate>2012-05-12T15:00:47</TestDate>
      <Status>Success</Status>
      <TotalFailures>0</TotalFailures>
      <TotalSuccess>2</TotalSuccess>
      <TotalIgnored>0</TotalIgnored>
      <Time />
      <Message>Test successful

Execution time: 0.78ms</Message>
    </UnitTestResult>
  </Results>
  <Tests>
    <TestRecord Name="PasswordUsher">
      <Results>
        <UnitTestResult>
          <TestDate>2012-05-12T15:00:36</TestDate>
          <Status>Success</Status>
          <TotalFailures>0</TotalFailures>
          <TotalSuccess>2</TotalSuccess>
          <TotalIgnored>0</TotalIgnored>
          <Time />
          <Message>Test successful

Execution time: 0.80ms</Message>
        </UnitTestResult>
        <UnitTestResult>
          <TestDate>2012-05-12T15:00:47</TestDate>
          <Status>Success</Status>
          <TotalFailures>0</TotalFailures>
          <TotalSuccess>2</TotalSuccess>
          <TotalIgnored>0</TotalIgnored>
          <Time />
          <Message>Test successful

Execution time: 0.78ms</Message>
        </UnitTestResult>
      </Results>
      <Tests>
        <TestRecord Name="Core">
          <Results>
            <UnitTestResult>
              <TestDate>2012-05-12T15:00:36</TestDate>
              <Status>Success</Status>
              <TotalFailures>0</TotalFailures>
              <TotalSuccess>2</TotalSuccess>
              <TotalIgnored>0</TotalIgnored>
              <Time />
              <Message>Test successful

Execution time: 0.80ms</Message>
            </UnitTestResult>
            <UnitTestResult>
              <TestDate>2012-05-12T15:00:47</TestDate>
              <Status>Success</Status>
              <TotalFailures>0</TotalFailures>
              <TotalSuccess>2</TotalSuccess>
              <TotalIgnored>0</TotalIgnored>
              <Time />
              <Message>Test successful

Execution time: 0.77ms</Message>
            </UnitTestResult>
          </Results>
          <Tests>
            <TestRecord Name="Tests">
              <Results>
                <UnitTestResult>
                  <TestDate>2012-05-12T15:00:36</TestDate>
                  <Status>Success</Status>
                  <TotalFailures>0</TotalFailures>
                  <TotalSuccess>3</TotalSuccess>
                  <TotalIgnored>0</TotalIgnored>
                  <Time />
                  <Message>Test successful

Execution time: 0.79ms</Message>
                </UnitTestResult>
                <UnitTestResult>
                  <TestDate>2012-05-12T15:00:47</TestDate>
                  <Status>Success</Status>
                  <TotalFailures>0</TotalFailures>
                  <TotalSuccess>3</TotalSuccess>
                  <TotalIgnored>0</TotalIgnored>
                  <Time />
                  <Message>Test successful

Execution time: 0.77ms</Message>
                </UnitTestResult>
              </Results>
              <Tests>
                <TestRecord Name="DataAccessTests">
                  <Results>
                    <UnitTestResult>
                      <TestDate>2012-05-12T15:00:36</TestDate>
                      <Status>Success</Status>
                      <TotalFailures>0</TotalFailures>
                      <TotalSuccess>5</TotalSuccess>
                      <TotalIgnored>0</TotalIgnored>
                      <Time />
                      <Message>Test successful

Execution time: 0.78ms</Message>
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2012-05-12T15:00:47</TestDate>
                      <Status>Success</Status>
                      <TotalFailures>0</TotalFailures>
                      <TotalSuccess>5</TotalSuccess>
                      <TotalIgnored>0</TotalIgnored>
                      <Time />
                      <Message>Test successful

Execution time: 0.75ms</Message>
                    </UnitTestResult>
                  </Results>
                  <Tests>
                    <TestRecord Name="can_add_a_service_record">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:36</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.18ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:47</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.17ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                    <TestRecord Name="can_delete_a_service_record">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:36</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.40ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
Connecting to: main
Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:47</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.41ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
Connecting to: main
Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                    <TestRecord Name="can_get_all_service_records">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:36</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.01ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:47</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.01ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                    <TestRecord Name="can_update_a_service_record">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:36</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.06ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:00:47</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.06ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                  </Tests>
                </TestRecord>
                <TestRecord Name="EncryptionTests">
                  <Results>
                    <UnitTestResult>
                      <TestDate>2012-05-12T15:00:36</TestDate>
                      <Status>Success</Status>
                      <TotalFailures>0</TotalFailures>
                      <TotalSuccess>1</TotalSuccess>
                      <TotalIgnored>0</TotalIgnored>
                      <Time />
                      <Message>Test successful

Execution time: 0.00ms</Message>
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2012-05-12T15:00:47</TestDate>
                      <Status>Success</Status>
                      <TotalFailures>0</TotalFailures>
                      <TotalSuccess>1</TotalSuccess>
                      <TotalIgnored>0</TotalIgnored>
                      <Time />
                      <Message>Test successful

Execution time: 0.00ms</Message>
                    </UnitTestResult>
                  </Results>
                  <Tests>
                    <TestRecord Name="can_encrypt_text">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T15:11:38</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.03ms</Message>
                          <ConsoleOutput>Original:   Lazy brown fox jumped over the fence.
Round Trip: Lazy brown fox jumped over the fence.
Error:   Expected: True
  But was:  False

</ConsoleOutput>
                          <ConsoleError />
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                  </Tests>
                </TestRecord>
                <TestRecord Name="AccountDataAccessTests">
                  <Results />
                  <Tests>
                    <TestRecord Name="can_add_an_account">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T16:49:31</TestDate>
                          <Status>Failure</Status>
                          <TotalFailures>1</TotalFailures>
                          <TotalSuccess>0</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>System.NotImplementedException : The requested feature is not implemented.</Message>
                          <StackTrace>  at PasswordUsher.Core.AccountDataAccess.Insert (PasswordUsher.Domain.Entities.Account entity) [0x00000] in /home/chirdeep/Projects/password-usher/PasswordUsher/PasswordUsher.Core/Data/AccountDataAccess.cs:16 
  at PasswordUsher.Core.Tests.AccountDataAccessTests.can_add_an_account () [0x00025] in /home/chirdeep/Projects/password-usher/PasswordUsher/PasswordUsher.Core.Tests/AccountDataAccessTests.cs:24 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                          <ConsoleOutput />
                          <ConsoleError />
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T16:51:51</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.23ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T16:55:09</TestDate>
                          <Status>Failure</Status>
                          <TotalFailures>1</TotalFailures>
                          <TotalSuccess>0</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Mono.Data.Sqlite.SqliteException : Abort due to constraint violation
columns ProviderId, Name are not unique</Message>
                          <StackTrace>  at Mono.Data.Sqlite.SQLite3.Reset (Mono.Data.Sqlite.SqliteStatement stmt) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SQLite3.Step (Mono.Data.Sqlite.SqliteStatement stmt) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteDataReader.NextResult () [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteDataReader..ctor (Mono.Data.Sqlite.SqliteCommand cmd, CommandBehavior behave) [0x00000] in &lt;filename unknown&gt;:0 
  at (wrapper remoting-invoke-with-check) Mono.Data.Sqlite.SqliteDataReader:.ctor (Mono.Data.Sqlite.SqliteCommand,System.Data.CommandBehavior)
  at Mono.Data.Sqlite.SqliteCommand.ExecuteReader (CommandBehavior behavior) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteCommand.ExecuteNonQuery () [0x00000] in &lt;filename unknown&gt;:0 
  at Dapper.SqlMapper.ExecuteCommand (IDbConnection cnn, IDbTransaction transaction, System.String sql, System.Action`2 paramReader, System.Object obj, Nullable`1 commandTimeout, Nullable`1 commandType) [0x00000] in &lt;filename unknown&gt;:0 
  at Dapper.SqlMapper.Execute (IDbConnection cnn, System.String sql, System.Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) [0x00000] in &lt;filename unknown&gt;:0 
  at PasswordUsher.Core.AccountDataAccess.Insert (PasswordUsher.Domain.Entities.Account entity) [0x00006] in /home/chirdeep/Projects/password-usher/PasswordUsher/PasswordUsher.Core/Data/AccountDataAccess.cs:19 </StackTrace>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T16:55:27</TestDate>
                          <Status>Failure</Status>
                          <TotalFailures>1</TotalFailures>
                          <TotalSuccess>0</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Mono.Data.Sqlite.SqliteException : Abort due to constraint violation
columns ProviderId, Name are not unique</Message>
                          <StackTrace>  at Mono.Data.Sqlite.SQLite3.Reset (Mono.Data.Sqlite.SqliteStatement stmt) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SQLite3.Step (Mono.Data.Sqlite.SqliteStatement stmt) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteDataReader.NextResult () [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteDataReader..ctor (Mono.Data.Sqlite.SqliteCommand cmd, CommandBehavior behave) [0x00000] in &lt;filename unknown&gt;:0 
  at (wrapper remoting-invoke-with-check) Mono.Data.Sqlite.SqliteDataReader:.ctor (Mono.Data.Sqlite.SqliteCommand,System.Data.CommandBehavior)
  at Mono.Data.Sqlite.SqliteCommand.ExecuteReader (CommandBehavior behavior) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteCommand.ExecuteNonQuery () [0x00000] in &lt;filename unknown&gt;:0 
  at Dapper.SqlMapper.ExecuteCommand (IDbConnection cnn, IDbTransaction transaction, System.String sql, System.Action`2 paramReader, System.Object obj, Nullable`1 commandTimeout, Nullable`1 commandType) [0x00000] in &lt;filename unknown&gt;:0 
  at Dapper.SqlMapper.Execute (IDbConnection cnn, System.String sql, System.Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) [0x00000] in &lt;filename unknown&gt;:0 
  at PasswordUsher.Core.AccountDataAccess.Insert (PasswordUsher.Domain.Entities.Account entity) [0x00006] in /home/chirdeep/Projects/password-usher/PasswordUsher/PasswordUsher.Core/Data/AccountDataAccess.cs:19 </StackTrace>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T21:32:16</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.19ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                    <TestRecord Name="can_get_all_accounts">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T16:55:02</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.14ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                    <TestRecord Name="can_update_an_account">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T16:57:33</TestDate>
                          <Status>Failure</Status>
                          <TotalFailures>1</TotalFailures>
                          <TotalSuccess>0</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>System.NotImplementedException : The requested feature is not implemented.</Message>
                          <StackTrace>  at PasswordUsher.Core.AccountDataAccess.Update (PasswordUsher.Domain.Entities.Account entity) [0x00000] in /home/chirdeep/Projects/password-usher/PasswordUsher/PasswordUsher.Core/Data/AccountDataAccess.cs:31 
  at PasswordUsher.Core.Tests.AccountDataAccessTests.can_update_an_account () [0x0001a] in /home/chirdeep/Projects/password-usher/PasswordUsher/PasswordUsher.Core.Tests/AccountDataAccessTests.cs:41 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T17:00:03</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.23ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                    <TestRecord Name="can_delete_an_account">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T17:02:33</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.51ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
Connecting to: main
Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                      <Tests />
                    </TestRecord>
                    <TestRecord Name="can_get_by_provider">
                      <Results>
                        <UnitTestResult>
                          <TestDate>2012-05-12T21:28:54</TestDate>
                          <Status>Failure</Status>
                          <TotalFailures>1</TotalFailures>
                          <TotalSuccess>0</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>System.InvalidOperationException : Database is not open</Message>
                          <StackTrace>  at Mono.Data.Sqlite.SqliteCommand.InitializeForReader () [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteCommand.ExecuteReader (CommandBehavior behavior) [0x00000] in &lt;filename unknown&gt;:0 
  at Mono.Data.Sqlite.SqliteCommand.ExecuteDbDataReader (CommandBehavior behavior) [0x00000] in &lt;filename unknown&gt;:0 
  at System.Data.Common.DbCommand.ExecuteReader () [0x00000] in &lt;filename unknown&gt;:0 
  at System.Data.Common.DbCommand.System.Data.IDbCommand.ExecuteReader () [0x00000] in &lt;filename unknown&gt;:0 
  at Dapper.SqlMapper+&lt;QueryInternal&gt;d__13`1[PasswordUsher.Domain.Entities.Account].MoveNext () [0x00000] in &lt;filename unknown&gt;:0 
  at System.Linq.Enumerable.Count[Account] (IEnumerable`1 source) [0x00000] in &lt;filename unknown&gt;:0 
  at PasswordUsher.Core.Tests.AccountDataAccessTests.can_get_by_provider () [0x0000f] in /home/chirdeep/Projects/password-usher/PasswordUsher/PasswordUsher.Core.Tests/AccountDataAccessTests.cs:57 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T21:32:01</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.16ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T21:38:38</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.16ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                        <UnitTestResult>
                          <TestDate>2012-05-12T21:38:51</TestDate>
                          <Status>Success</Status>
                          <TotalFailures>0</TotalFailures>
                          <TotalSuccess>1</TotalSuccess>
                          <TotalIgnored>0</TotalIgnored>
                          <Time />
                          <Message>Test successful

Execution time: 0.17ms</Message>
                          <ConsoleOutput />
                          <ConsoleError>Connecting to: main
</ConsoleError>
                        </UnitTestResult>
                      </Results>
                    </TestRecord>
                  </Tests>
                </TestRecord>
              </Tests>
            </TestRecord>
          </Tests>
        </TestRecord>
      </Tests>
    </TestRecord>
  </Tests>
</TestRecord>